content/for-individuals/index.zh.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "个体"
+date: 2018-03-20T08:09:08+13:00
+draft: false
+type: "default"
+author: ""
+description: ""
+---
+{{< hero_section
+titleText="欢迎来到 NavCoin"
+paragraphText="如果您是这个项目的新手，并且不确定从哪里开始，那么这里有一些方便的步骤可以&nbsp;遵循。"
+imgSrc="/images/for-beginners/beg-hero.svg"
+>}}
+{{< step_container_section 
+    titleText="Getting Started"
+    subtitleText=""
+>}}
+  {{< step 
+      titleText="第一步: 了解NavCoin"
+      imgSrc="/images/for-beginners/beg-learn.svg"
+      text=" NavCoin;一种经济实惠、快捷环保的加密货币。自2014年以来，围绕NavCoin已经形成了一个完整的生态系统，从易于使用的移动钱包到简化的私人支付解决方案——所有这些都是由一个活跃的国际社会建立和维护的，没有中央权力或&nbsp;治理。"
+      buttonText="阅读更多"
+      buttonUrl="/why-navcoin"
+      buttonImgSrc="/images/icons/rightward-arrow.svg"
+  >}}
+  {{< step 
+      titleText="第二部: 选择您的钱包"
+      imgSrc="/images/for-beginners/beg-wallets.svg"
+      text="您的钱包是与您的数字硬币交互的主要方式，也是您用来发送和接收导航的。有不同类型的钱包——从能让你完全控制并参与NavCoin网络的“full”钱包，到更容易使用的“thin”&nbsp;钱包。"
+      buttonText="钱包"
+      buttonUrl="/wallets"
+      buttonImgSrc="/images/icons/rightward-arrow.svg"
+  >}}
+  {{< step
+      titleText="第三部：获取NAV"
+      imgSrc="/images/for-beginners/beg-get-nav.svg"
+      text="一旦你选择了你的钱包，有几个方法来获得导航。其中最快的是通过加密货币交换。你会把BTC发送到交易所，然后把它转换成NAV。你可以在一些顶级交易所找到NAV，比如Binance, Bittrex和&nbsp;Poloniex。"
+      buttonText="购买NavCoin"
+      buttonUrl="/buy-navcoin"
+      buttonImgSrc="/images/icons/rightward-arrow.svg"
+  >}}
+  {{< step
+      titleText="第四部：安全的网络"
+      imgSrc="/images/for-beginners/beg-secure-network.svg"
+      text="如果您正在运行NavCoin核心钱包，那么您可以选择加注您的硬币，并获得每年4%的加注奖励。有一系列的设备可以使标记变得更容易，并且社区创建了许多信息来帮助您&nbsp;设置。"
+  >}}
+  {{< step
+      titleText="第5步：花费 NavCoin"
+      imgSrc="/images/for-beginners/beg-spend.svg"
+      text="你可以用你的NavCoin在网上和商店里买各种各样的东西。看看这些接受导航和显示你的支持的&nbsp;商家。"
```

Comment:
This section sounds like it has been google translated or needs improving

Reply:
fixed